packages/vulcan-forms/lib/components/FormSubmit.jsx

```diff
@@ -48,7 +50,12 @@ const FormSubmit = ({
       </a>
     ) : null}
   
-    {deleteDocument ? (
+    { deleteDocument && Users.canDelete({
+        context: { Users },
```

Comment:
Thanks for the update! Are you sure you need to pass the context here?

Reply:
I wasn't sure, I copied it from Card.jsx where canUpdate was called with this parameter, but I guess it was for backwards compatibility which we don't need in this case. I fixed it.